bowtie/tests/test_benchmarks.py

```diff
@@ -359,6 +350,59 @@ async def test_benchmark_run_markdown_output(
         assert expected_data1 in stdout
         assert expected_data2 in stdout
 
+    @pytest.mark.asyncio
+    async def test_benchmark_run_varying_param_markdown(
+        self,
+        tmp_path,
+    ):
+        from bowtie.tests.benchmarks import benchmark_with_varying_parameter
+
+        tmp_path.joinpath("benchmark.json").write_text(
+            json.dumps(
+                benchmark_with_varying_parameter.get_benchmark().serializable(),
+            ),
+        )
+        stdout, stderr = await bowtie(
+            "perf",
+            "-i",
+            DIRECT_CONNECTABLE,
+            "-q",
+            "--format",
+            "markdown",
+            tmp_path / "benchmark.json",
+            exit_code=0,
+        )
+
+        expected_data1 = """
```

Comment:
Can you use `textwrap.dedent` to make these a bit less "ugly" indented in a follow-up PR? Will merge for now.